cmake/compiler/arcmwdt/target.cmake

```
@@ -63,3 +63,8 @@ endif()
 # The MWDT compiler doesn't need to pass any properties to the linker as for now
 function(compiler_set_linker_properties)
 endfunction()
+
+# Include architecture-specific settings
+if("${ARCH}" STREQUAL "riscv")
+  include(${CMAKE_CURRENT_LIST_DIR}/target_riscv.cmake)
+endif()
```

Hint: You can also do like this and thereby allow to just ad new `target_<arch>.cmake` files when needed without having to add extra `if() ... elseif() ... endif()` in future.
```suggestion
include(${CMAKE_CURRENT_LIST_DIR}/target_${ARCH}.cmake OPTIONAL)
```